### ISSUE TITLE
In a fresh vagrant box, "make" fails

### ISSUE THREAD

**Issue by robparrott** [flynn-archive/flynn-demo#9]
Error is as follows:

```
vagrant@packer-virtualbox:/vagrant$ make
sudo bin/godeb install 1.2
processing https://go.googlecode.com/files/go1.2.linux-amd64.tar.gz
package go_1.2-godeb1_amd64.deb ready
Selecting previously unselected package go.
(Reading database ... 55012 files and directories currently installed.)
Unpacking go (from go_1.2-godeb1_amd64.deb) ...
Setting up go (1.2-godeb1) ...
go get -v github.com/flynn/go-discover/discoverd
bitbucket.org/kardianos/osext
github.com/coreos/go-systemd/journal
github.com/coreos/go-log/log
github.com/coreos/go-etcd/etcd
github.com/flynn/rpcplus
github.com/flynn/rpcplus/jsonrpc
github.com/flynn/rpcplus/comborpc
github.com/flynn/go-discover/discover
# github.com/flynn/go-discover/discover
src/github.com/flynn/go-discover/discover/backend_etcd.go:32: response.Kvs undefined (type *etcd.Response has no field or method Kvs)
src/github.com/flynn/go-discover/discover/backend_etcd.go:57: undefined: etcd.KeyValuePair
src/github.com/flynn/go-discover/discover/backend_etcd.go:98: b.Client.GetAll undefined (type *etcd.Client has no field or method GetAll)
src/github.com/flynn/go-discover/discover/backend_etcd.go:98: not enough arguments to return
src/github.com/flynn/go-discover/discover/backend_etcd.go:103: b.Client.WatchAll undefined (type *etcd.Client has no field or method WatchAll)
src/github.com/flynn/go-discover/discover/backend_etcd.go:117: too many arguments in call to b.Client.Get
src/github.com/flynn/go-discover/discover/backend_etcd.go:122: b.Client.CompareAndSwap undefined (type *etcd.Client has no field or method CompareAndSwap)
src/github.com/flynn/go-discover/discover/backend_etcd.go:122: resp.Value undefined (type []*etcd.Response has no field or method Value)
src/github.com/flynn/go-discover/discover/backend_etcd.go:122: resp.ModifiedIndex undefined (type []*etcd.Response has no field or method ModifiedIndex)
src/github.com/flynn/go-discover/discover/backend_etcd.go:122: too many errors
make: *** [bin/discoverd] Error 2
```

**Comment by titanous** [comment]
Run `go get -v -u github.com/flynn/go-discover/discoverd` to get the latest discoverd, I fixed that yesterday with flynn/go-discover@39dfc9f2dd2e016bb58e4ee148a8ba95a02eccce.

**Comment by robparrott** [comment]
That seems to have resolved that issue.
